src/navigation/authStack.tsx

```diff
@@ -1,12 +1,38 @@
-import { NavigationContainer } from "@react-navigation/native";
-import { createStackNavigator } from "@react-navigation/stack";
+import React from 'react';
```

Comment:
Changes in this screen should probably be in a separate PR, since they're completely unrelated.